app/controllers/main/routes.py

```diff
@@ -11,3 +12,15 @@ def home():
         #TODO We have to return some sort of error page
         print('Error in main page:', e)
         return "",500
+
+@main_bp.route('/volunteerIndicateInterest', methods = ['GET'])
+def volunteerIndicateInterest():
+    programs = Program.select()
+    interests = Interest.select().where(Interest.user_id == g.current_user)
```

Comment:
use `Interest.user` instead of `Interest.user_id`. Let's avoid referencing ids directly wherever we can